R/style_theme.R

```diff
@@ -3,6 +3,14 @@
 #' @description Create a simple ggplot theme for graphs
 #'
 #' @details We can change this later to accomodate opinions...
+#'
+#' @export
+#'
+#' @example  Create a scatterplot with style!
+#'           ggplot2::ggplot(iris,aes(Sepal.Width,Sepal.Length,
+#'           color=Species))+
+#'           geom_point(size=5)+
+#'           style_theme()
 
 style_theme <- function(){
   theme_bw() %+replace%
```

Comment:
Whoa, cool.

In terms of scoping, where is the plot coming from in this case? Why isn't it passed as an argument to the function? Just asking.